pkg/backend/file/file.go

```diff
@@ -67,6 +67,29 @@ func (f *Wrapper) Size() (int64, error) {
 	return stat.Size(), nil
 }
 
+func (f *Wrapper) GetRevisionCounterDisabled() (bool, error) {
```

Comment:
`IsRevisionCounterDisabled` or `IsRevisionCounterEnabled`.

Reply:
Hi Sheng,
At beginning I was named it exactly the same as you suggested, but after I think about it may return error, not only a boolean, I changed it to get. Please let me know what you think. I just feel a little wired if we named it 'IsRevisionCounterDisabled', and it also may return an error. Thanks.

Reply:
It's fine for `IsRevisionCounterDisabled` to return an error. `Is` indicates it's a boolean, which is better suited here.